lnbits/wallets/cliche.py

```diff
@@ -1,10 +1,12 @@
 import asyncio
 import hashlib
 import json
+from asyncio import Future
 from typing import AsyncGenerator, Optional
 
 from loguru import logger
-from websocket import create_connection
+from websocket import WebSocketApp
+from websockets.client import connect
```

Comment:
we do we need 2 different libraries?

Reply:
it's a WIP! websockets to try async, `WebSocketApp` has hability to pass `on_message` method. will fix

Reply:
While I understand that using 2 different libraries is a problem, i think websockets is more suitable that websocket in a coroutine app like fastapi as websockets library provides coroutines natively